docs/Developers/Git_Basics.rst

```diff
@@ -453,3 +453,9 @@ use an interactive rebasing session. This opens an editor where you can enter
 commands (described below) for each commit to be rebased. These commands
 determine how individual commits will be transferred to the new base. You can
 also reorder the commit listing to change the order of the commits themselves.
+
+If you would like more information/commands, please use your favourite search
+engine to look for it. You could also read the
+`git documentation <https://git-scm.com/doc>`_ to learn more. Git is widely used
```

Comment:
I disagree with the addition of a link. The point of this issue was to enable the user to look for things on their own, thus I don't think we should hand them a link to what they're looking for. Making them search for something teaches them that they need to search for things. They won't search for anything if we give them a link.
Opinions?